packages/ir-sdk/fern/apis/ir-types-latest/definition/types.yml

```diff
@@ -116,10 +116,11 @@ types:
       types: list<SingleUnionType>
       baseProperties: list<ObjectProperty>
   SingleUnionType:
-    extends: commons.WithDocs
+    extends: commons.Declaration
```

Comment:
thoughts on changing this from a declaration to just adding the availability directly to the extends list? might make it a bit more clear that while this isn't a type declaration the addition of availability is intentional